.travis.yml

```diff
@@ -3,10 +3,20 @@ rvm:
 - 2.2.2
 gemfile:
 - spec/dummy/Gemfile
-install: bundle install
+install:
+- bundle install
+- cd spec/dummy/client && npm install
+- $(npm bin)/webpack --config webpack.server.js
+- $(npm bin)/webpack --config webpack.client.js
+env:
+ - export RAILS_ENV=test
+before_script:
+- cd ../../
+- export DISPLAY=:99.0
+- sh -e /etc/init.d/xvfb start
 script:
 - rake run_rspec:gem
-- rake run_rspec:dummy
+- DRIVER=selenium_firefox rake run_rspec:dummy
```

Comment:
@justin808 because this is for Travis CI, and that is what they use. Pretty much all CI's use firefox browser for the selenium web driver.